src/main/host/descriptor/socket/inet/udp.rs

```diff
@@ -898,22 +898,11 @@ impl UdpSocket {
                 self.recv_buffer
                     .set_soft_limit_bytes(val.try_into().unwrap());
             }
-            (libc::SOL_SOCKET, libc::SO_REUSEADDR) => {
-                // TODO: implement this, tor and tgen use it
-                log::warn!("setsockopt SO_REUSEADDR not yet implemented");
-            }
-            (libc::SOL_SOCKET, libc::SO_REUSEPORT) => {
-                // TODO: implement this, tgen uses it
-                log::warn!("setsockopt SO_REUSEPORT not yet implemented");
-            }
-            (libc::SOL_SOCKET, libc::SO_KEEPALIVE) => {
-                // TODO: implement this, libevent uses it in
-                // evconnlistener_new_bind()
-                log::warn!("setsockopt SO_KEEPALIVE not yet implemented");
-            }
             (libc::SOL_SOCKET, libc::SO_BROADCAST) => {
                 // TODO: implement this, pkg.go.dev/net uses it
-                log::warn!("setsockopt SO_BROADCAST not yet implemented");
+                log::warn!(
+                    "setsockopt SO_BROADCAST not yet implemented for udp; ignoring but returning 0"
+                );
             }
             _ => {
```

Comment:
Somewhat orthogonal from this PR, but this should probably also use `warn_once_then_debug`

Reply:
Added.